go/vt/vtgate/vstream_manager.go

```diff
@@ -408,6 +411,21 @@ func (vs *vstream) alignStreams(ctx context.Context, event *binlogdatapb.VEvent,
 	}
 }
 
+func (vs *vstream) getCells() []string {
+	var cells []string
+	if vs.optCells != "" {
+		for _, cell := range strings.Split(strings.TrimSpace(vs.optCells), ",") {
+			cells = append(cells, strings.TrimSpace(cell))
+		}
+	}
```

Comment:
Wouldn't this one liner be the same?
```
cells := strings.Split(strings.TrimSpace(vs.optCells), ",")
```

Seems that way: https://go.dev/play/p/WwiNpEnhwsJ

IMO that's easier to read and it should be more efficient too.

Reply:
Ah, good point 🙂 you’re right then